etc/firefox-common-addons.inc

```diff
@@ -53,3 +53,12 @@ whitelist ${HOME}/.wine-pipelight
 whitelist ${HOME}/.wine-pipelight64
 whitelist ${HOME}/.zotero
 whitelist ${HOME}/dwhelper
+
+# GNOME Shell integration (chrome-gnome-shell) needs dbus and python 3 (blacklisted by disable-interpreters.inc)
```

Comment:
you should move the above commented `noblacklist ${HOME}/.local/share/gnome-shell/extensions` and its associated whitelist option down here